### ISSUE TITLE
Default version number and switching version number: weird behavior

### ISSUE THREAD

**Issue by amsimon** [react-hook-form/documentation#845]
Navigating to https://react-hook-form.com , the default version is v5/v6. Shouldn't it be v7? Also, switching the version number doesn't seem to be working properly. Probably related to the redirect issue referenced in this pull request https://github.com/react-hook-form/documentation/pull/843

https://user-images.githubusercontent.com/5640304/195208199-93374463-5418-40ff-9329-cc14d158d019.mov

**Comment by bluebill1049** [comment]
you can click on the API page, it should show the correct doc: https://legacy.react-hook-form.com/v6/api

**Comment by amsimon** [comment]
Ok, but the version select is still not behaving correctly. 

- Why does clicking the "V5/V6" button open a new tab with "Version 7" selected?
- Why does clicking "Version 7" open a dropdown where I can select "Version 6" or "Version 5" but I can't select these versions when I go to https://react-hook-form.com?

**Comment by bluebill1049** [comment]
Yes, there is something to be improved here for the transition to the old section. I have tagged enhancement for this issue.
